aiohttp/web_fileresponse.py

```diff
@@ -20,17 +20,16 @@
 
 class SendfilePayloadWriter(PayloadWriter):
 
-    def set_transport(self, transport):
-        self._transport = transport
-
-        if self._drain_waiter is not None:
-            waiter, self._drain_maiter = self._drain_maiter, None
-            if not waiter.done():
-                waiter.set_result(None)
+    def __init__(self, *args, **kwargs):
+        self.__buffer = []
```

Comment:
It's mostly taste question but why don't invite new name for this buffer instead of mangling?
We never use mangling in aiohttp.